javascript/javascript3/week3/shoppingcart/shoppingCart.js

```diff
@@ -0,0 +1,110 @@
+const container = document.getElementById('mainPage')
```

Comment:
I cannot get the shoppingcart to work. It will show the HTML, but no products and search doesnt work.